### ISSUE TITLE
Custom Adjustments.

### ISSUE THREAD

**Issue by atapatel** [vanilophp/adjustments#1]
Hello, Is it possible to add custom adjustments like.
If a user buys 1 iPhone, we give them AirPods free.

**Comment by fulopattila122** [comment]
Yes, it was designed to achieve such functionality. See the **Adjusters** section here: https://vanilo.io/docs/master/adjustments

Please note that this module is still WIP. To be released in December/January 2022.
